lighthouse-cli/test/smokehouse/test-definitions/source-maps/expectations.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+
+const mapPath = require.resolve('../../../fixtures/source-map/script.js.map');
+const mapJson = fs.readFileSync(mapPath, 'utf-8');
+const map = JSON.parse(mapJson);
+
+/**
+ * @type {Array<Smokehouse.ExpectedRunnerResult>}
+ * Expected Lighthouse audit values for seo tests
+ */
+const expectations = [
+  {
+    artifacts: {
+      SourceMaps: [
+        {
+          scriptUrl: 'http://localhost:10200/source-map/source-map-tester.html',
+          sourceMapUrl: 'http://localhost:10200/source-map/script.js.map',
+          map,
```

Comment:
if we do proceed with the cookie thing, would love to see a samesite cookie set on our server and have it throw a 403 or something if it gets requested with samesite + a particular query string or something

Reply:
I'll give it a shot.

btw, removing this source map smoke - legacy-javascript covers this fetching maps.